assets/css/style.css

```diff
@@ -289,6 +289,36 @@ html[dir='rtl'] #srcLanguages {
     margin-left: 30px;
 }
 
+/* SpellCheker */
+
+#spellcheckerInput {
+    display: block;
+    height: 131px;
+    overflow: auto;
+    resize: both;
+}
+
+.spellcheckVisible .spellError {
+    border-bottom: 1px solid #f00;
+}
+
+.popover-content { /* sass-lint:disable-line class-name-format */
+    max-height: 190px;
+    overflow: auto;
+    padding: 0;
+    width: 110%;
+}
+
+.list-group { /* sass-lint:disable-line class-name-format */
```

Comment:
Same with this and the next two. Basically my ask is to wrap it in a more specific class.